### PR TITLE
Revert Module.prepend and replace alias_method_chain with alias_method

### PR DESCRIPTION
**Pull request by pschambacher**
:octopus::computer:

/cc @grosser @bquorning 

### Description
Waiting for https://bugs.ruby-lang.org/issues/11120 to be fixed, we'll do with shitty code...

### Steps to merge
* [ ] :+1: from the team

### Risks
* None

### CODE REVIEW

**Comment by bquorning** [comment]
:+1: :cry:

File: lib/active_record_host_pool/connection_adapter_mixin.rb [hunk 1]
```diff
@@ -7,41 +7,53 @@
 
 module ActiveRecordHostPool
   module DatabaseSwitch
-    def self.prepended(base)
+    def self.included(base)
       base.class_eval do
         attr_accessor(:_host_pool_current_database)
+
+        alias_method :execute_without_switching, :execute
```
Comment (grosser) [inline lib/active_record_host_pool/connection_adapter_mixin.rb]: just 1 alias ?

**Comment by grosser** [comment]
:+1:

**Comment by bquorning** [comment]
> just 1 alias ?

What do you mean @grosser ?

**Comment by grosser** [comment]
1 alias instead of 2 per method

**Comment by bquorning** [comment]
There are 2 aliases already. Do you suggest using just 1, and if so, how?

**Comment by pschambacher** [comment]
I think I understand. Michael suggest to move the first alias before the method and use a simple monkey patch rather than naming the method _with_something

**Comment by grosser** [comment]
yes!

```
alias_method :foo_without_bar, :foo
def foo
  bar
  foo_without_bar
end

```

**Comment by bquorning** [comment]
The 2-alias approach is tested and works. I’d appreciate a separate PR.

**Pull request merged**